app/src/main/java/com/keylesspalace/tusky/di/StorageModule.kt

```diff
@@ -65,8 +66,11 @@ object StorageModule {
                 AppDatabase.MIGRATION_41_42, AppDatabase.MIGRATION_42_43, AppDatabase.MIGRATION_43_44,
                 AppDatabase.MIGRATION_44_45, AppDatabase.MIGRATION_45_46, AppDatabase.MIGRATION_46_47,
                 AppDatabase.MIGRATION_47_48, AppDatabase.MIGRATION_52_53, AppDatabase.MIGRATION_54_56,
-                AppDatabase.MIGRATION_58_60
+                AppDatabase.MIGRATION_58_60, AppDatabase.MIGRATION_58_60
```

Comment:
```suggestion
                AppDatabase.MIGRATION_58_60, AppDatabase.MIGRATION_60_62
```

Reply:
Oops, I had to redo this merge when I was cleaning up the history and made a mistake. Good catch.